app/src/main/java/fr/gaulupeau/apps/Poche/service/IntentServiceBase.java

```diff
@@ -61,6 +62,11 @@ protected ActionResult processException(Exception e, String scope) {
                         && e.getMessage().contains("Connection timed out")) {
                     result.setErrorType(ActionResult.ErrorType.TEMPORARY);
                     handled = true;
+                } else if(e instanceof java.net.SocketException
+                        && e.getMessage() != null
+                        && e.getMessage().contains("Software caused connection abort")) {
```

Comment:
i am not a 100% sure, but comparing text seems instable and not the correct way to catch exactly this case. 
Is the error message independent from any language settings?

Reply:
are there different types of SocketException or why do you want only the one with that specific string in it?

Reply:
I just scrolled some lines in this file up and saw we were handling other cases live this, too.

Reply:
No, nothing to change. I was just too fast with writing down my thoughts. I should have inspected the source code more.